src/main/java/core/basesyntax/service/impl/CsvDataReaderServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.DataReaderService;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvDataReaderServiceImpl implements DataReaderService {
```

Comment:
I do not see anything CSV-specific in this implementation. So, why do you have Csv in class name?